source/screens/LoginScreen.js

```diff
@@ -112,6 +164,23 @@ function LoginScreen(props) {
     }, [isAuthenticated, navigateToScreen])
   );
 
+  /**
+   * Effect for showing notification if an error occurs
+   */
+  useEffect(() => {
+    if (isRejected && error?.message) {
+      showNotification(error.message, '', 'error');
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [error]);
+
+  /**
+   * Effect for disabling login button if personal number is not required length
+   */
+  useEffect(() => {
+    setIsButtonDisabled(personalNumber.length !== 12);
+  }, [isButtonDisabled, personalNumber]);
```

Comment:
Very minor thing, but this seems a bit unnecessary, the isButtonDisabled is only used in one place, and at that place, you can just put the check directly,  see the other comment.